osm2gtfs/creators/ni_esteli/ni_esteli.json

```diff
@@ -30,6 +30,6 @@
         "version":  "0.1"
     },
     "schedule_source": "https://github.com/mapanica/pt-data-esteli/raw/master/timetable.json",
-    "output_file": "data/ni-esteli.zip",
-    "selector": "esteli"
+    "output_file": "data/ni-esteli-gtfs.zip",
+    "selector": "ni_esteli"
```

Comment:
Why did you go for `_` in the selector's case? I think it would be easier and more consistent to use `-` here (as in the output file, for example) too.

Reply:
Because in python module names underscores are allowed, according to [PEP8](https://www.python.org/dev/peps/pep-0008/), but hyphens aren't. To me it seems much more consistent to use the underscore, because all files and the creator directory only can contain underscores and no hyphens. Anyway the output file can be specified as wished, there is no rule here.

Reply:
OK, I understand, thanks for your explanation!